common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -84,10 +164,14 @@ public static String getLocalHostName(int timeout) {
   }
 
   /**
+   * Check if the underlying OS is Windows.
+   */
+  public static final boolean WINDOWS = System.getProperty("os.name").startsWith("Windows");
```

Comment:
There's an apache `SystemUtils.IS_OS_WINDOWS` variable that might be safer to use.